marklogic-data-hub/src/main/resources/hub-internal-config/security/roles/data-hub-developer.json

```diff
@@ -6,7 +6,12 @@
     "manage-user",
     "ps-user",
     "rest-admin",
-    "tde-admin"
+    "tde-admin",
```

Comment:
@srinathgit I just added the requirement to 3617 for flow-developer-role to get these roles as well, and for flow-operator-role to get all the reader roles.

Reply:
Added reader roles to "flow-operator-role" and writer roles to "flow-developer-role"